src/interface/TWElrondTransactionFactory.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
```

Comment:
Consider adding the following:
```suggestion

// This file was created by the Elrond Network (www.elrond.com)
```

Reply:
External contributions have to use the same file header.